{"body": "\nIt's a shame we'll never know what caused this problem, whether it was\nthe model or a bug....\n\n\n\n", "replyTo": "Subject: Problem solved somehow. (Re: [Radiance-general] instance problem?)", "sender": "Gregory J. Ward", "isquestion": false, "tags": [], "id": "<cf92157d86c8cc87c02ac995d75fec3b@gmail.com>", "refs": ["<4d24fd59a4510d3e08db30fed037a4d8@gmx.net>", "<339bed7366992175463110acada8f351@gmail.com>", "<ce937a6db6b5f1b9c10a528b8fc63b59@gmx.net>", "<7df7a15b4d0162d47059ec6d73774152@gmail.com>", "<c29b54ba081fed6cb4534b11838f8aa0@gmx.net>", "<4f08ac6912446393dfee0e64c48bf9ef@gmx.net>"], "datetime": "Mon Apr 18 18:15:15 2005", "email": "gregoryjward at gmail.com", "subject": "Problem solved somehow. (Re: [Radiance-general] instance problem?)"}